src/main.rs

```diff
@@ -243,9 +231,9 @@ async fn run(error_sender: Sender<anyhow::Error>) -> Result<()> {
 	let last_full_node_ws = data::get_last_full_node_ws_from_db(db.clone())?;
 
 	let version = rpc::Version {
-		version: "1.6.0".to_string(),
+		version: cfg.full_node_rpc_version.clone(),
```

Comment:
This shouldn't be configuration since its compile time dependency. But, since we are using our avail client to communicate with the node, maybe specifying dependency is not needed any more (its implicitly specified with avail-subxt version)?

Reply:
Agree! I think avail-sbuxt is enough to guarantee compatibility right now.